docs/source/docker_archive/advanced/active_learning.rst

```diff
@@ -3,6 +3,12 @@
 Active Learning
 ===============
 
+.. warning::
+    **The Docker Archive documentation is deprecated**
+
+    The old workflow described in these docs will not be supported with new Lightly Worker versions above 2.6.
+    Please switch to our `new documentation page <https://docs.lightly.ai>`_ instead.
```

Comment:
This looks great and should cover the most important pages! One small suggestion. Let's link to `https://docs.lightly.ai/docs` instead. Then they are directly on the right page.

Reply:
I changed it.